oci/defaults.go

```diff
@@ -119,6 +119,7 @@ func DefaultLinuxSpec() specs.Spec {
 			"/proc/timer_list",
 			"/proc/timer_stats",
 			"/proc/sched_debug",
+			"/sys/firmware",
```

Comment:
Isn't this already added back on line 209? https://github.com/cyphar/docker/blob/7cac2ac7cbe53f2a57bfdf434c2e275752718416/oci/defaults.go#L209-L212

Reply:
(may warrant a code comment here though)

Reply:
Oh, oops. :wink: